test/Driver/driver-time-compilation.swift

```diff
@@ -0,0 +1,7 @@
+// RUN: %swiftc_driver -driver-time-compilation %s 2>&1 | %FileCheck %s
```

Comment:
We do have special logic for when there's only one command to perform, down at the bottom of Compilation.cpp. I think if `-driver-time-compilation` is passed, we should skip that, just like if we're supposed to produce parseable output.